common/pinot/pinotQueryValidator_test.go

```diff
@@ -301,6 +301,14 @@ func TestValidateQuery(t *testing.T) {
 			validated: "",
 			err:       "failed to process a bool key to SQLVal: &{<nil> abc { }}",
 		},
+		"case22-1: test not equal to a string field": {
```

Comment:
Could you add cases
* CustomStringField is missing
* CustomStringFieldA != 'abc' AND CustomStringFieldB != 'def'
* CustomStringFieldA = 'abc' AND CustomStringFieldB != 'def'
* CustomStringFieldA != 'abc' AND CustomStringFieldB = 'def'
* CustomStringFieldA = 'abc' AND CustomStringFieldB = 'def'
* CustomStringFieldA != 'abc' OR CustomStringFieldB != 'def'
* CustomStringFieldA = 'abc' OR CustomStringFieldB != 'def'
* CustomStringFieldA != 'abc' OR CustomStringFieldB = 'def'
* CustomStringFieldA = 'abc' OR CustomStringFieldB = 'def'